### PR TITLE
Extend PVLAN documentation

### PR DESCRIPTION
**Pull request by nvazquez**
Extend PVLAN documentation as per changes introduced by PR: https://github.com/apache/cloudstack/pull/3732 

FS: https://cwiki.apache.org/confluence/display/CLOUDSTACK/Private+VLAN+%28PVLAN%29+support+on+Layer+2+networks

### CODE REVIEW

**Comment by nvazquez** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
LGTM in general.

But would be nice to explain things with pictures from gui as examples, or alternatively with API - I'm talking about the last few lines explaining how you can create different kind of PVLAN setups (primisc. vs isolated vs community)

**Comment by andrijapanicsb** [comment]
requesting docbuild

File: source/adminguide/networking/isolation_in_advanced_zone_with_vlan.rst [hunk 1]
```diff
@@ -77,23 +80,34 @@ Prerequisites
...
 
--  **Secondary Isolated VLAN ID**:
+-  **Secondary Isolated VLAN ID**: The PVLAN ID to use within the primary VLAN.
+
+-  **PVLAN Type**: The PVLAN type corresponding to the PVLAN ID to use within the primary VLAN.
+
+Creating a PVLAN-enabled network can be done in multiple ways depending on the PVLAN type:
+
+   - For a **promiscuous** PVLAN:
+      - Set the secondary VLAN ID to the same VLAN ID as the primary VLAN that the promiscuous PVLAN will be inside, or
```
Comment (andrijapanicsb) [inline source/adminguide/networking/isolation_in_advanced_zone_with_vlan.rst:102]:
```suggestion
      - Set the secondary VLAN ID to the same VLAN ID as the primary VLAN that the promiscuous PVLAN will be inside (available only via API, not UI), or
```

File: source/adminguide/networking/isolation_in_advanced_zone_with_vlan.rst [hunk 2]
```diff
@@ -77,23 +80,34 @@ Prerequisites
...
+      - Set the PVLAN type to "Promiscuous" and do not set the secondary VLAN ID.
 
-   - For a **promiscuous** PVLAN, set this to the same VLAN ID as the primary VLAN
-     that the promiscuous PVLAN will be inside.
-   - For an **isolated** PVLAN, set this to the PVLAN ID which you wish to use
-     inside the primary VLAN.
+   - For an **isolated** PVLAN:
+      - Set the secondary VLAN ID to the PVLAN ID which you wish to use inside the primary VLAN, or
```
Comment (andrijapanicsb) [inline source/adminguide/networking/isolation_in_advanced_zone_with_vlan.rst:106]:
```suggestion
      - Set the secondary VLAN ID to the PVLAN ID which you wish to use inside the primary VLAN (available only via API, not UI), or
```

**Pull request merged**